### DISCUSSION TITLE
Error Try Republisihing Your App on Chrome 71

### DISCUSSION THREAD

**Discussion by hci-ikahera** [Budibase/budibase#10507] [Help] [answered]
Hi everyone, I want to ask you about Chrome minimum version for running Budibase App. I have found that on Chrome version 71, I could not open the app. Here is the error,

<img width="1010" alt="MicrosoftTeams-image (11)" src="https://user-images.githubusercontent.com/95213000/236735436-7e710826-2de4-4a70-bb9f-c85f1310732d.png">

Anybody knows what cause the error? Is it related to Chrome minimum version or there is something else?
Thank you.

Regards,
Ika H.

**Comment by andz-bb** [comment, marked as answer]
@hci-ikahera hey Ika, I have done some testing and I get the same result on Chromium 71. I am running the old x86 builds under emulation on an ARM Mac. It's not the most stable but it works enough to test. The main error I see in the console is a syntax error.
<img width="1512" alt="chrome 71 error" src="https://github.com/Budibase/budibase/assets/110921612/d7d9ba6a-011a-40da-ab26-3c192e4933f7">

The line where it is breaking is using the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) which is a new part of JavaScript syntax introduced in ES11 (ECMAScript 2020). Chrome 71 is from 2018 and does not support these new JavaScript features that Budibase relies on. The first version of Chrome that supports this new syntax is version 80. I have tested it very briefly to verify this, and in Chromium 80 my app loads past that error screen and takes me to my app. 

<img width="1270" alt="chrome 80 working" src="https://github.com/Budibase/budibase/assets/110921612/23f2c94f-f533-4035-9e26-8a2832868da6">

Because Chrome 80 itself is also so old, I can't make any guarantees about Budibase's functionality on it beyond just loading into the app. We would always recommend using the most recent update for any given browser, especially for security reasons. I would strongly encourage the user to update their Chrome version if it's at all possible.

**Reply by melohagan** [reply]
Great answer, and yes keep your browser up-to-date 😁 
It's important for security as well!

**Comment by hci-ikahera** [comment]
Oh, I see. Thank you for you detail explanation @andz-bb 🙏🏼
